test/unit/transactionRunner-test.js

```diff
@@ -298,11 +290,7 @@ describe('TransactionRunner', () => {
       beforeEach(() => {
         transaction.response.status = 400;
         transaction.origin.filename = filename;
-
-        if (!runner.configuration.data) { runner.configuration.data = {}; }
-        if (!runner.configuration.data[filename]) { runner.configuration.data[filename] = {}; }
-        runner.configuration.data[filename].mediaType = 'text/plain';
-
+        transaction.apiDescriptionMediaType = 'text/plain';
```

Comment:
Shouldn't this be a Swagger media type?

> when processing other than OpenAPI 2 document and given transaction has non-2xx status code

Reply:
"other than OpenAPI 2 document", so it should be anything else than Swagger

Reply:
Oh my bad, misread.